yellowbrick/target/binning.py

```diff
@@ -0,0 +1,176 @@
+
+# yellowbrick.target.binning
+# Implementations of histogram with vertical lines to help with balanced binning.
+#
+# Author:   Juan L. Kehoe (juanluo2008@gmail.com)
+# Author:   Prema Damodaran Roman (pdamo24@gmail.com)
+
+# Created:  Tue Mar 13 19:50:54 2018 -0400
+#
+# Copyright (C) 2018 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: binning.py
+
+"""
+Implements histogram with vertical lines to help with balanced binning.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+import matplotlib.pyplot as plt
+import numpy as np
+
+from .base import TargetVisualizer
+from yellowbrick.exceptions import YellowbrickValueError
+from yellowbrick.utils import is_dataframe
```

Comment:
Also remove this line to prevent [Unused Import error](https://travis-ci.com/DistrictDataLabs/yellowbrick/jobs/136370433#L1868) during checking.